src/__tests__/snapshottests.test.js

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
+import Home from '../components/home';
+import Quote from '../components/quote';
+
+describe('Snapshot for Calculator and Home', () => {
+  it('Snapshot should match component render', () => {
+    const homeview = render(<Home />);
+    expect(homeview).toMatchSnapshot();
+  });
+  it('Snapshot should match component render', () => {
+    const quoteview = render(<Quote />);
+    expect(quoteview).toMatchSnapshot();
+  });
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=js%20using%20Jest.-,Create%20unit%20tests%20for%20all%20React%20components%3A,Use%20Jest%20snapshots%20to%20test%20the%20components.,-Use%20React%20Testing), Please add screenshot tests for the `Calculator` and `navbar` components as you did with `home` and `quote` components.